api/valor_api/backend/metrics/text_generation.py

```diff
@@ -214,18 +214,16 @@ def _setup_llm_client(
     if "client" in llm_api_params and "api_url" in llm_api_params:
         raise ValueError("Cannot specify both client and api_url.")
 
-    client = llm_api_params.get("client")
-    if client is not None:
-        if client == "openai":
+    client_name = llm_api_params.get("client")
+    if client_name is not None:
+        if client_name == "openai":
```

Comment:
nit: this would be a use case for `match case` instead of multiple if statements